### PR TITLE
[sites-showcase] Adding target for external links

### PR DESCRIPTION
**Pull request by mattbag**
Personally I am used to showcase sites links to open a new tab (definitely for github pages that I might open and read later), also the icon on visit site is suggesting new tab ;)
I skipped the url under the title, and added a space between the icon and 'visit'.

Also note that upon closing this window, the sidebar filters are not retained. I think it's a proper bug.

### CODE REVIEW

**Comment by LeKoArts** [comment]
When addind „_blank“ you also need to add the „noopener noreferrer“ rel tag

**Comment by fk** [comment]
I personally am for not opening new windows/letting the user make that decision.
That said I can see how it makes sense doing so here, but…I would still stick to my general rule. ;-)
I don't want to make the call though! :-P

I think @LeKoArts might be in favor of opening a new window here (judging from the footer links on your [website](https://www.lekoarts.de/))?

What do you think @m-allanson @pieh?

**Comment by mattbag** [comment]
Alright just to be clear, the issue with filter selection loss is upon opening the popup for the site. 
Since it's 'routing away' from the list view. I didn't have time to check properly how this 2 views are linked, but keeping the param shouldn't be hard.
If this is done intentionally never mind. 
I see the _blank matter is a subjective topic so I am happy to see more feedback coming.

File: www/src/components/showcase-details.js [hunk 1]
```diff
@@ -470,6 +474,7 @@ const ShowcaseDetails = ({ parent, data, isModal, categories }) => (
                         verticalAlign: `sub`,
                       }}
                     />
+                    {` `}
                     Visit site
                     {` `}
```
Comment (agarrharr) [inline www/src/components/showcase-details.js]: Why put spaces around " Visit site "?

Comment (mattbag) [reply www/src/components/showcase-details.js]: The one on the left, was there, so I didn't feel like removing, the one before is to give icon some breathing.

**Comment by mattbag** [comment]
Someone else is working on this and other improvements. 
I don't mind dropping, but opinions on dev usage for links are still appreciated

**Comment by KyleAMathews** [comment]
I agree with @fk, let's people choose to open items in a new tab/window vs. us forcing it.

Thanks for the idea @mattbag!

**Pull request closed without merging**